src/components/TodoContainer.js

```diff
@@ -2,38 +2,94 @@ import React from 'react';
 import style from "./TodoContainer.module.css";
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
+import PropTypes from 'prop-types';
 
 
-const AIRTABLE_API_URL = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
 
-function TodoContainer() {
+const AIRTABLE_API_URL = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
 
-    const [todoList, setTodoList] = React.useState([])
-    const [isLoading, setIsLoading] = React.useState(true)
+function TodoContainer({ tableName }) {
+    const [todoList, setTodoList] = React.useState([]);
+    const [isLoading, setIsLoading] = React.useState(true);
+    const [sortBy, setSortBy] = React.useState('asc');
 
     const fetchData = React.useCallback(async () => {
-
         const options = {
             method: 'GET',
             headers: { Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}` }
         }
+        // const sortDirection = isAscending ? 'asc' : 'desc';
+        const AIRTABLE_API_URL = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${tableName}?view=Grid%20view&sort[0][field]=title&sort[0][direction]=asc`;
```

Comment:
Nice choice of using the Airtable API to do the sorting for your application. When possible do the sorting outside of your client application, ideally on the way in or in the backend. This will greatly help performance as your application grows.